### DISCUSSION TITLE
How to submit additions to template documentation?

### DISCUSSION THREAD

**Discussion by twinotter** [arendst/Tasmota#10207] [Q&A] [answered]
Hello!

I'm confused about how to add to the template documentation. I can't edit it directly (as makes sense). The Issues section seems like the most straight forward approach, but it says it's only for broken links and problems. And I'm also not even sure if the project is collecting the kind of information I have.

In this case, I got an Aoycocr X10S to work through serial programming. I want to share images of the module and my approach.

I do notice that this kind of information isn't in many of the template sections... so perhaps there is somewhere else this is placed? If so, I don't see any links to it nor did it come up in a google search.

Thanks!
Rudy

**Comment by Jason2866** [comment, marked as answer]
https://templates.blakadder.com/aoycocr_X10S.html  and click the **EDIT ON GITHUB**
after doing this click the *pencil* (doing this creates a fork) do your changes and submit a PR (Pull Request)

**Reply by twinotter** [reply]
Thanks for this! I'll go that route. How do I upload images?
